test/unit/SimulateOrder.t.sol

```diff
@@ -34,7 +34,7 @@ contract SimulateOrder_Unit_Test is BaseTest {
         uint256 token1PoolReserve = 3000e18; // 3000 USDC
         // Set prices
         uint256 price0 = 300e8; // 300 USD/ETH
-        uint256 price1 = 1e8; // 1 ETH/USD
+        uint256 price1 = 1e8; // 1 USD/USDC
```

Comment:
This notation is about prices being relative/denominated in the other. Keeping the ratio notation might not hurt, but I'm not opposed to losing it.